content/en/docs/tasks/debug-application-cluster/crictl.md

```diff
@@ -41,27 +37,36 @@ of Kubernetes. Extract it and move it to a location on your system path, such as
 The `crictl` command has several subcommands and runtime flags. Use
 `crictl help` or `crictl <subcommand> help` for more details.
 
-`crictl` connects to `unix:///var/run/dockershim.sock` by default. For other
-runtimes, you can set the endpoint in multiple different ways:
+You can set the endpoint for `crictl` by doing one of the following:
 
-- By setting flags `--runtime-endpoint` and `--image-endpoint`
-- By setting environment variables `CONTAINER_RUNTIME_ENDPOINT` and `IMAGE_SERVICE_ENDPOINT`
-- By setting the endpoint in the config file `--config=/etc/crictl.yaml`
+* Set the `--runtime-endpoint` and `--image-endpoint` flags.
+* Set the `CONTAINER_RUNTIME_ENDPOINT` and `IMAGE_SERVICE_ENDPOINT` environment
+  variables.
+* Set the endpoint in the configuration file `--config=/etc/crictl.yaml`.
```

Comment:
Is this also accurate:
```suggestion
* Set the endpoint in a configuration file and then specify the path to that file
  when you run `crictl`. For example: `crictl --config=/etc/crictl.yaml <subcommand>`.
```
?

Reply:
Normally: Set the endpoint in the configuration file `/etc/crictl.yaml`. Unless you want to move it ?